scripts/publish-documentation.sh

```diff
@@ -35,6 +35,7 @@ mkdir -p "./$FOLDER"
 mv -v $OUTPUT/* "./$FOLDER"
 
 step "Switching branch to publisher-production"
+git checkout Gemfile.lock
```

Comment:
Should we remove `--depth=1 --prune` from `git fetch --depth=1 --prune` above _(line 17)_?

Reply:
Yes, please remove the shallow clone. I always avoided the publish-documentation.sh script because this step trashes my clone.